Chinese/Javascript/js.mdown

```diff
@@ -0,0 +1,3 @@
+在Js开发的路上漫游。
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.